<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideMprisController</class>
 <widget class="QWidget" name="SlideMprisController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="mprisSelection">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="arrow-down"/>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mprisIcon">
        <property name="text">
         <string notr="true">MusicIconLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="mprisTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string notr="true">Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mprisMetadata">
          <property name="text">
           <string notr="true">Extras</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="mprisBack">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-skip-backward">
          <normaloff>../../../../tsscreenlock/screenlock</normaloff>../../../../tsscreenlock/screenlock</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mprisPlay">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-playback-start">
          <normaloff>../../../../tsscreenlock/screenlock</normaloff>../../../../tsscreenlock/screenlock</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mprisNext">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-skip-forward">
          <normaloff>../../../../tsscreenlock/screenlock</normaloff>../../../../tsscreenlock/screenlock</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
